app/views/hosts/select_multiple_puppet_ca_proxy.html.erb

```diff
@@ -0,0 +1,12 @@
+<%= render 'selected_hosts', :hosts => @hosts %>
+
+<%= form_for :puppet, :url => update_multiple_puppet_ca_proxy_hosts_path(:host_ids => params[:host_ids]) do |f| %>
```

Comment:
Nearly the same logic as `select_multiple_puppet_proxy.html.erb`, extract to a shared partial or helper